README.md

```diff
@@ -56,6 +59,20 @@ Access web locally:
 Check `Makefile` for shell commands if you want to run them with modified
 parameters.
 
+### Local .env file
+
+Local `.env` file is required to run the application, a basic example of such file contains the following:
```

Comment:
Technically, some component's don't require any configuration. So I would change it like this (just apply the suggestion if you agree)...
```suggestion
Creating a local `.env` file (see [Docker docs](https://docs.docker.com/compose/env-file/)
on `.env` files) is necessary for some components in the app to work correctly,
a basic example of such file contains the following:
```